imports/ui/App.vue

```diff
@@ -5,20 +5,38 @@
         <component :is="Component"/>
       </transition>
     </router-view>
-    <alert-message/>
-    <loader/>
+    <alert-message ref="alertMessage"/>
+    <loader ref="loader"/>
   </v-app>
 </template>
 
 <script lang="ts">
 import AlertMessage from '@components/Utilities/Alerts/AlertMessage.vue';
 import Loader from '@components/Utilities/Loaders/Loader.vue';
-import { defineComponent } from 'vue';
+import { defineComponent, onMounted, provide, ref } from 'vue';
+import { Injections } from '@typings/utilities';
+import mitt from 'mitt';
 
 export default defineComponent({
   name: 'App',
   components: { AlertMessage, Loader },
+  setup() {
+    const alertMessage = ref(null);
+    const loader = ref(null);
+
+    onMounted(() => {
+      provide(Injections.AlertMessage, alertMessage.value);
+      provide(Injections.Loader, loader.value);
+      const emitter = mitt();
+      provide(Injections.Emitter, emitter);
+    })
```

Comment:
what is this for? 

if it is to use those components in the global scope, I had implemented in their mounted hooks the code to make them globals. Besides, I added mitt in the `imports/startup/client/index.ts` file to configure the event emitter as global.